documentation/1.9/content/userguide/tools/create.md

```diff
@@ -32,7 +32,7 @@ To create more complex domains, it may be necessary to create a custom domain ty
 
 ### Using an encrypted model
 
-If the model or variables file contains passwords encrypted with the WDT Encryption tool, decrypt the passwords during create with the `-use_encryption` flag on the command line to tell the Create Domain Tool that encryption is being used and to prompt for the encryption passphrase.  As with the database passwords, the tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input.
+If the model or variables file contains passwords encrypted with the WDT Encryption tool, decrypt the passwords during create with the `-use_encryption` flag on the command line to tell the Create Domain Tool that encryption is being used and to prompt for the encryption passphrase.  As with the database passwords, the tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input. You can bypass the stdin prompt with two other options. Store the passphrase in an environment variable, and use the environment variable name with command line option -passphrase_env. Another option is to create a file with the single value of passphrase. Provide the name of the file with command line option -passphrase_file. The passphrase will be read by the tool from the file. 
```

Comment:
Another option is to create a file with the single value of passphrase. -> Another option is to create a file with the value for the passphrase. 
All similar sentences below should use the same text.   @rosemarymarano any suggestions?

Reply:
I would re-word these sentences as follows, "You can bypass the stdin prompt with two other options: store the passphrase in an environment variable, and use the environment variable name with the command-line option `-passphrase_env` or create a file with the single value of the passphrase. Provide the name of the file with the command-line option `-passphrase_file`. The passphrase will be read by the tool from the file.